ci_framework/roles/os_must_gather/tasks/main.yml

```diff
@@ -46,7 +46,8 @@
           oc adm must-gather --image {{ cifmw_os_must_gather_image }}
           --timeout {{ cifmw_os_must_gather_timeout }}
           --host-network={{ cifmw_os_must_gather_host_network }}
-          --dest-dir {{ cifmw_os_must_gather_output_dir }}/logs > {{ cifmw_os_must_gather_output_dir }}/logs/os_must_gather.log
+          --dest-dir {{ cifmw_os_must_gather_output_dir }}/logs
+          -- SOS_SERVICES= gather > {{ cifmw_os_must_gather_output_dir }}/logs/os_must_gather.log
```

Comment:
Hi @fmount, I think there is a typo here and it should be `--SOS_SERVICES=gather` rather than `-- SOS_SERVICES= gather`

Reply:
Here https://paste.centos.org/view/raw/391e519e is the output and README command: https://github.com/openstack-k8s-operators/openstack-must-gather/blob/main/README.md?plain=1#L36

Reply:
no it's good :D
`SOS_SERVICES` it's not a cli variable. It's an environment variable passed to the must-gather command, so it's good to have the space to properly invoke the `gather` script.